shared/teams/main/team-list.js

```diff
@@ -38,9 +48,15 @@ const Row = ({name, membercount, onOpenFolder, onManageChat, onViewTeam}: RowPro
           <Text type="BodySemibold">
             {name}
           </Text>
-          <Text type="BodySmall">
-            {membercount + ' member' + (membercount !== 1 ? 's' : '')}
-          </Text>
+          <Box style={globalStyles.flexBoxRow}>
+            {isNew &&
```

Comment:
we have a little metadata tag component in common that should likely be used
https://github.com/keybase/client/blob/master/shared/common-adapters/meta.js.flow